### DISCUSSION TITLE
Video informativo sobre: FORMATO DE ESPECIFICACIÓN DE REQUERIMIENTOS DE SOFTWARE

### DISCUSSION THREAD

**Discussion by FabioMurillo16** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#53] [Ideas]
Hola Explorers aquí les comparto un video sobre la practica de requerimientos, explicando como se llena este formato.
https://www.youtube.com/watch?v=AotyBHVKp8I

**Comment by KarenPamelaLopezMedina** [comment]
> Hola Explorers aquí les comparto un video sobre la practica de requerimientos, explicando como se llena este formato. https://www.youtube.com/watch?v=AotyBHVKp8I

Muchas gracias!!
